actors/verifreg/src/types.rs

```diff
@@ -66,9 +65,10 @@ pub struct RemoveDataCapReturn {
     pub data_cap_removed: DataCap,
 }
 
-#[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq)]
-#[serde(transparent)]
-pub struct RemoveDataCapProposalID(pub u64);
+#[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
```

Comment:
If you kept the type as a tuple struct and change the serialization directives to `Serialize_tuple, Deserialize_tuple` as you've done over the field struct I would expect this to serialize correctly.  Did you try this?

Reply:
@ZenGround0 I did, but the macros aren't properly setup for this to work. I wound up with things complaining that `Serialize` wasn't implemented for `RemoveDataCapProposalID`.

I didn't try very hard to get it to work.